com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeBrickPool.cs

```diff
@@ -140,8 +146,8 @@ internal void Allocate(int numberOfBrickChunks, List<BrickChunkAlloc> outAllocat
             {
                 if (m_NextFreeChunk.z >= m_Pool.depth)
                 {
-                    Debug.Assert(false, "Cannot allocate more brick chunks, probevolume brick pool is full.");
-                    break; // failure case, pool is full
+                    Debug.LogError("Cannot allocate more brick chunks, probe volume brick pool is full.");
```

Comment:
Ok for now, but we should remember to remove the error when streaming is in. Pool full should just mean that a load/unload operation needs to happen.

Reply:
Yup, that's the plan. Just wanted to have something explicit in the meantime